source/lambda/documentbulkprocessor/lambda_function.py

```diff
@@ -58,26 +59,37 @@ def uploadDocument(bucketName, documentKey, filename, s3Client):
 
     # open local file
     datafile = open('/tmp/' + filename, 'rb')
-    
+
     if datafile == None:
         print("UploadDocument: failed to open local file: " + filename)
         return False
-    
+
     # upload local file to s3
     s3Client.upload_fileobj(datafile, bucketName, documentKey)
     return True
 
 
+def is_invalid_mime_type(filename):
```

Comment:
Can we  move this to the lambda_helper rather than having the functioned defined twice?

Reply:
I thought about that, but it means adding a new zip file and build command into the build system, which is also something i wanted to avoid. The helper lambda is currently just a python file.